apps/prairielearn/src/ee/pages/instructorAiGenerateDrafts/instructorAiGenerateDrafts.html.ts

```diff
@@ -46,6 +46,21 @@ export function InstructorAIGenerateDrafts({
             >
               AI Generation Draft Questions
               <div class="d-flex flex-row">
+                ${drafts.length > 0
+                  ? html`
+                      <form method="POST" class="mr-2">
+                        <input
+                          type="hidden"
+                          name="__csrf_token"
+                          value="${resLocals.__csrf_token}"
+                        />
+                        <button class="btn btn-sm btn-light" name="__action" value="delete_drafts">
+                          <i class="fa fa-trash" aria-hidden="true"></i>
+                          <span class="d-none d-sm-inline">Delete all drafts</span>
+                        </button>
```

Comment:
We should use a confirmation modal for a destructive action like this.

Reply:
Added in fd0f052ad2ce1e73b49a4680c35f09c149548574